docs/geedocs/5.3.3/sources/answer/releaseNotes/relNotesGEE5_3_3.rst.txt

```diff
@@ -1,8 +1,6 @@
 
 Release notes: Open GEE 5.3.3
```

Comment:
I think this is an auto-generated file. I believe that the `.rst` file is the source.

Reply:
Fixed